lidur 1 kóði

Thordisalda opened the pull request:
liður 1 ætti allur að vera klár i þessari skrá, endilega fyrir yfir og hvort þið fáið sömu niðurstöður og ég þegar þið keyrið :)

Thordisalda committed:
lidur 1 kóði

liður 1 ætti allur að vera klár i þessari skrá, endilega fyrir yfir :)

IngvarAtli commented:
Ég fékk sömu niðrstöður þegar ég keyrði kóðann, virkar vel. Kóðinn er líka mjög vel uppsettur og snyrtilegur.

IngvarAtli commented:
Þegar Logi er búinn að fara yfir má mergea

Thordisalda committed:
Textaútskýring á liður 1

Útskýri hvað við erum að gera í kóðanum fyrir lið 1 og hverjar niðurstöðurnar eru.

Jon-slow97 reviewed (approved):
Glæsilegt, virkaði hjá mér og mjög áhugaverðar niðurstöður.

IngvarAtli reviewed (commented):
Mjög flott eins og kom fram í comments hér í conversations

IngvarAtli reviewed (approved):
þarna kom approve
